app/assets/src/stylesheets/04-templates/templates.scss

```diff
@@ -0,0 +1,22 @@
+header {
+  @include module-colors;
+  margin: 0;
+  padding: 1rem 2rem;
+}
+
+main {
+  @include module-colors;
+  border-radius: 1rem;
+  padding: 1rem;
+  width: 100%;
+  width: calc(100% - 2rem);
```

Comment:
Not a typo; doubled width rule is meant to accommodate browsers that don't support `calc`. Since `calc` has 94% global usage this may be overkill. Idk.